src/examples/PopupExamplePage.tsx

```diff
@@ -1,6 +1,13 @@
 'use strict';
-import {Text, TouchableHighlight, View} from 'react-native';
-import React, {useState, useRef} from 'react';
+import {
+  AccessibilityInfo,
```

Comment:
nit: remove the unused AccessibilityInfo, TextInput, and findNodeModuleHandle imports